CloudFormationTemplate.yml

```diff
@@ -0,0 +1,205 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >
+  This CloudFormation template creates simple Lambda functions,
+  which prints CloudFormation resource Arn from the stack.  
+Resources:
+  LambdaFunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service:
+              - lambda.amazonaws.com
+          Action:
+            - sts:AssumeRole
+      Path: "/"
+      Policies:
+      - PolicyName: LambdaFunctionPolicy
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action:
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+              - dynamodb:scan
+              - dynamodb:list
+              - s3:ListAllMyBucket
+              - s3:ListBucket
+              - s3:PutObject
+              - s3:PutObjectAcl
+              - s3:GetBucketLocation
+            Resource: 
+            - arn:aws:dynamodb:eu-west-2:251045990954:table/MetadataJson
+            - arn:aws:lambda:eu-west-2:251045990954:function:Stack-10-LambdaFunction-HUcUFXbCq4ZJ
+            - arn:aws:s3:::dbmetadata/
+            - '*'
+  LambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: python3.7
+      Timeout: 5
+      Handler: index.lambda_handler
+      Role: !GetAtt LambdaFunctionRole.Arn
+      Code:
+        ZipFile:
+          !Sub
+            - |-
+              import boto3
+              from datetime import date, datetime
+              import csv
+              def lambda_handler(event, context):
+                  dynamodb = boto3.resource("dynamodb", region_name='eu-west-2')
+                  table = dynamodb.Table('MetadataJson')
+                  
+                  tableDict = table.scan(
+                      ProjectionExpression='#AN, account, #S',
+                      ExpressionAttributeNames={
+                          '#AN': 'account-name',
+                          '#S': 'status',
+                      },
+                  )
+                  
+                  
+                  tableList = list(tableDict.items())
+                  tableResponse = tableList[0][1]
+                  tableData = []
+                  for i in range(len(tableResponse)):
+                          tableData.append([tableResponse[i].get('account-name'), 
+                          tableResponse[i].get('account'), 
+                          tableResponse[i].get('status')])
+                          
+                          
+                  csv_name = 'Metadata-'+str(date.today())+'-'+datetime.now().strftime("%H-%M")+'.csv'
```

Comment:
please use f-strings instead of + for concatenation 
https://peps.python.org/pep-0498/